school_library.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Getters
+  attr_reader :name
+
+  attr_reader :age
```

Comment:
- According to the project requirements, the :name, :age variable should have getters and setters, while the :id should have only a getter, please fix this 🙏

Reply:
I changed the form using the short hand, but forgot to delete these lines. :handshake: Thanks for your point of clarification :clap:

Reply:
- [ ] As mentioned by the previous reviewer, since name and age are both supposed to have getters and setters, ill recommend setting them as `attr_accessor`